core/cloudflow-akka/src/main/scala/cloudflow/akkastream/AkkaStreamletContextImpl.scala

```diff
@@ -36,24 +39,86 @@ import org.apache.kafka.clients.consumer.ConsumerConfig
 import org.apache.kafka.clients.producer.ProducerRecord
 import org.apache.kafka.common.serialization._
 import cloudflow.streamlets._
+import org.slf4j.LoggerFactory
 
 import scala.concurrent.duration.{ DurationInt, FiniteDuration }
 
 /**
  * Implementation of the StreamletContext trait.
  */
+@InternalApi
 final class AkkaStreamletContextImpl(
     private[cloudflow] override val streamletDefinition: StreamletDefinition,
     sys: ActorSystem
 ) extends AkkaStreamletContext {
+  private val log                            = LoggerFactory.getLogger(classOf[AkkaStreamletContextImpl])
+  private val streamletDefinitionMsg: String = s"${streamletDefinition.streamletRef} (${streamletDefinition.streamletClass})"
 
   implicit val system: ActorSystem = sys
 
   override def config: Config = streamletDefinition.config
 
+  private val consumerStopTimeout: FiniteDuration =
+    if (config.hasPath("consumer-stop-timeout"))
+      FiniteDuration(config.getDuration("consumer-stop-timeout").toMillis, TimeUnit.MILLISECONDS).toCoarsest
+    else 5.seconds
```

Comment:
Where in the configs would the default need to be added?

Reply:
In the reference.conf of cloudflow-akka?

Reply:
That would work with an independent default.
Is there a place where the `consumer-stop-timeout` default would be propagated as a default to the streamlet configs? (Is there a config which all streamlet configs inherit from?)